qutip/tests/test_qubits.py

```diff
@@ -0,0 +1,28 @@
+from numpy.testing import assert_, run_module_suite
+from qutip.qip.qubits import qubit_states
+from qutip import basis, tensor
```

Comment:
The imports must be: `from qutip.states import basis` and `from qutip.tensor import tensor`